frappe/installer.py

```diff
@@ -809,7 +809,7 @@ def get_old_backup_version(sql_file_path: str) -> Version | None:
 	if match := re.search(r"Frappe (\d+\.\d+\.\d+)", header[0]):
 		backup_version = match[1]
```

Comment:
```suggestion
		return Version(match[1])
```

and can just `return None` outside, should be simpler

Reply:
That indeed looks simpler. Thank you.